src/main/java/com/answerdigital/academy/answerking/controller/CategoryController.java

```diff
@@ -25,7 +25,7 @@
 
 @Validated
 @RestController
-@RequestMapping(path = "/categories")
+@RequestMapping(path = "/api/categories")
```

Comment:
we don't have to change it in every method and class. The changes will be taken care in Bench-159 which will be 2-3 code config change.